packages/select/src/utils/themeConverter.ts

```diff
@@ -59,12 +59,13 @@ export const themeConverter = (
       ...getElementStyles('placeholder'),
       ...getInteractiveStyles('placeholder', isDisabled, isFocused, isSelected),
     }),
-
     group: (css) => ({
       ...css,
+      ...getElementStyles('group'),
     }),
     groupHeading: (css) => ({
       ...css,
+      ...getElementStyles('group'),
```

Comment:
It is strange that you put group styles here too.
If you need custom styles there it will be better to create `groupHeading` property in theme.

Reply:
I forgot to edit this prop 😱 I will do it in the next commit!